project.clj

```diff
@@ -1,9 +1,19 @@
-(defproject image-utils "0.1.0-SNAPSHOT"
-  :description "FIXME: write description"
+(defproject image-utils "0.1.0"
+  :description "PICI Image Utilities"
   :url "http://example.com/FIXME"
-  :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
-            :url "https://www.eclipse.org/legal/epl-2.0/"}
-  :dependencies [[org.clojure/clojure "1.10.0"]]
-  :main ^:skip-aot image-utils.core
+  :license {:name "GPL-3.0 "
```

Comment:
I haven't read enough about the various open source licenses to have a strong preference right now. I went with GPL on this because we're using GPL for Mantis. We're using GPL for Mantis because Federico is using it for his projects. I'm not tied to GPL at all, and am definitely open to a discussion about using a different license!